### ISSUE TITLE
Public Websites update request from Caregivers

### ISSUE THREAD

**Issue by ala-yna** [department-of-veterans-affairs/va.gov-team#7326]
### Background
- We are launching an online version of the 10-10CG form and need changes to the Program of
Comprehensive Assistance for Family Caregivers landing page support the form. This would ideally include a CTA link to the form and some content from the paper version of the form.

 * **Is the request related to a VA.gov tool? Provide a brief description of the tool or functionality:**  
   - The 10-10CG is an intake form for Veterans to apply for the Program for Comprehensive Assistance for Family Caregivers (PCAFC). Veterans and caregivers (primary and secondary) create a joint application that is routed to caregiver support coordinators to determine eligibility for the program.

* **Provide links to any available collateral: product briefs/outlines, wireframes, prototypes, user research synthesis:** 
   * [User Research Synthesis](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/caregivers/1010cg-mvp/usability-testing-jan2020/research-findings.md)
   * [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/caregiver/product-outline.md)
   * [Prototype](https://vsateams.invisionapp.com/share/P5WKZ1WQMXB) 

### Request Type
*Select all applicable and provide details behind your request* 
      
- [x] **Content support for a static public website page**
   * Is this to update existing static content? 
      * Yes
   * Does this need a new static tool landing page? (The content team may propose a different approach in consultation with IA.)
      * No
   * Provide a list of URL/s that need to be reviewed or updated. 
      *  https://www.va.gov/health-care/family-caregiver-benefits/comprehensive-assistance/
   * If it's a small copy section, list what the new info should be and what it is replacing.
       * List of qualifications required for caregiver eligibility
       * Update Veteran’s eligibility qualifications based on paper form
       * List of benefits available to eligible caregivers
       * List of materials of documents needed to apply
       * Link to new online 10-10CG form

- [x] **New URL or Redirect Requests**
   * Have you vetted this request by the VSP IA team?
      * Yes
   * Link to VSP IA Redirect Request ticket
      * https://github.com/department-of-veterans-affairs/va.gov-team/issues/6939
      

### Desired/target implementation date
*What is the ETA for the launch or date when request needs to be completed (for instance, for a user research session or eBenefits migration)? Our team's capacity will determine which sprint the work falls into.*

* May 29th "dark release"

### Requestor info
*Indicate what team and who the primary contacts are on your team* 

- Team name: **Caregivers**
- DEPO product owner: **CC Gong**
- Product manager: **Alayna Abel**
- Content writer: **Patrick Sanders**
- Designer: **Jonathan Nelson**
- FE engineer: **Justin Linn**
- Product/team Slack channel: **#vsa-caregivers**

--- 
#### Acceptance Criteria
- [x] Convert this request into an Epic
- [x] Follow-up with requestor to address any lingering questions/concerns
- [x] Create epic/tickets as needed for the Public Websites team to implement

#### Definition of Done
* Request ticket is closed and implementation tickets created as needed

**Comment by ala-yna** [comment]
Tagging: @psanders86, @jenniferlee-dsva, @sshein, @jonathanjnelson, @ccgongie for awareness.

**Comment by psanders86** [comment]
Tagging: @LouCarlozo63

**Comment by MickinSahni** [comment]
Hey @ala-yna, we'll need a draft of what the copy should be so that we can review for plain language, etc. 

Side note - there's a separate meeting with Caregiver stakeholders next week that *may or may not* coincide/overlap/supercede/etc. with this request. 

@jenniferlee-dsva, want to make sure from your end that the team isn't conflating these two things: the Caregiver request and other Mission Act initiatives that haven't been prioritized quite yet.

**Comment by sshein** [comment]
Hi @MickinSahni I think you and I and Laura and Alayna (and Jen Lee if available) should probably circle up. The caregiver product team is not going to provide copy. The idea is your team will work with the program team for strategy. Let's sync up.

**Comment by sshein** [comment]
My understanding is that there are two potential initiatives:

**Comment by sshein** [comment]
My understanding is that @jenniferlee-dsva  wants to meet with the caregiver stakeholders to discuss both parts of this project - the first part figuring out how we want to edit the current caregiver page on VA.gov https://www.va.gov/health-care/family-caregiver-benefits/comprehensive-assistance/ in order to incorporate language from their paper form and add a CTA to the online form experience. The second part related to the larger initiatives that haven't been prioritized. The Caregiver product team already has a few "suggestions" written for how to edit the Drupal content page, but it's up to Jen Lee and team at this point to decide what works in collaboration with the stakeholders. Those "suggestions" can be found at the end of this content doc.
[10-10CG-content.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/4428544/10-10CG-content.docx)

**Comment by LouCarlozo63** [comment]
@sshein I'm happy to jump in and provide assistance here. As there are many people already involved, my presence may only add to the complexity. But to the extent I can provide clarity and positivity, count me in. That includes my own time in any copy/copy editing flow.

**Comment by jenniferlee-dsva** [comment]
I don't know which comment to reply to first. 

Tl;dr - @AnneHurley is leading the content writing of the unauthenticated family caregiver program benefit page. @ Danielle (whom I can't seem to @ here yet with updated GH account) is DEPO content strategy. 

Initial call with key stakeholder/SMEs, provided to us by @mlighdc is set, where Anne and Danielle will introduce public web content team process and gather additional information to groom /break out content tickets. 

Per convo with Ming, there are possibly 3 aspects to updating the static content page: 

1/ Adding the tool widget and updating copy as needed for that launch. 
2/ Updating content page with immediate benefit information updates, related to MISSION Act changes to the program. 
3/ Determining what updates we need to be aware of - to slate for later in the fall, when full MISSION Act changes take effect. 

1 and 2 might happen together/same time. 

@AnneHurley and Danielle - please sync before caregiver call and share program knowledge. Let me know if you'd like me to join. We can also use Monday's weekly sync to devote to caregiver if no other items to cover.

**Comment by DanielleThierryUSDSVA** [comment]
Hey, @jenniferlee-dsva. I'm in Github now!

**Comment by AnneHurley** [comment]
@daniellethierryusdsva Trying this to see if I can tag you here - I'd love to chat with you about the ^ above before we meet on Wednesday with the Caregiver stakeholders. I would also like to understand if the VHIE / health records page/s are related to the MISSION act updates, which I believe they are. Let's chat.
@jenniferlee-dsva  FYSA @MickinSahni  FYSA

**Comment by DanielleThierryUSDSVA** [comment]
@AnneHurley Just verifying that notification came through. Thanks!

**Comment by jenniferlee-dsva** [comment]
From meeting with Ming - please include this in the tickets we'll break out for this work: 

Stakeholders identified for us by Ming are: 
•	Elsie Moore
•	Pam Wright
•	Cari Malcolm (optional)
•	Lisa Moore (as fysa only)
•	Jeff Grandon (or another VHA digital comms person assigned by Jeff, if Jeff can’t attend)


Related VA.gov benefit hub pages:
https://www.va.gov/health-care/family-caregiver-benefits/comprehensive-assistance/
https://www.va.gov/family-member-benefits/

Known legacy pages – it’s a morass, with no IA:
https://www.caregiver.va.gov/support/support_benefits.asp
https://www.va.gov/COMMUNITYCARE/index.asp
https://www.caregiver.va.gov/
https://www.mirecc.va.gov/coaching/
https://www.caregiver.va.gov/help_landing.asp

References: 
https://www.missionact.va.gov/
https://www.va.gov/opa/pressrel/pressrelease.cfm?id=5398

FYSA @daniellethierryusdsva and @AnneHurley @MickinSahni

**Comment by DanielleThierryUSDSVA** [comment]
Got it - thanks @jenniferlee-dsva and @mlighdc!
